test/data/test_crud_data.py

```diff
@@ -388,12 +388,49 @@ def test_get_by_id(self):
         data_object.get = mock_get
         data_object.get_by_id(uuid="UUID", additional_properties=["Test", "list"], with_vector=True)
         mock_get.assert_called_with(
-            uuid="UUID", class_name=None, additional_properties=["Test", "list"], with_vector=True
+            uuid="UUID",
+            class_name=None,
+            additional_properties=["Test", "list"],
+            with_vector=True,
+            node_name=None,
+            consistency_level=None,
         )
 
         data_object.get_by_id(uuid="UUID2", additional_properties=["Test"], with_vector=False)
         mock_get.assert_called_with(
-            uuid="UUID2", class_name=None, additional_properties=["Test"], with_vector=False
+            uuid="UUID2",
+            class_name=None,
+            additional_properties=["Test"],
+            with_vector=False,
+            node_name=None,
+            consistency_level=None,
+        )
+
+        data_object.get_by_id(
+            uuid="UUID3",
+            additional_properties=["Test"],
+            with_vector=False,
+            consistency_level="QUORUM",
+        )
+        mock_get.assert_called_with(
+            uuid="UUID3",
+            class_name=None,
+            additional_properties=["Test"],
+            with_vector=False,
+            node_name=None,
+            consistency_level="QUORUM",
+        )
+
+        data_object.get_by_id(
+            uuid="UUID4", additional_properties=["Test"], with_vector=False, node_name="node1"
+        )
+        mock_get.assert_called_with(
+            uuid="UUID4",
+            class_name=None,
+            additional_properties=["Test"],
+            with_vector=False,
+            node_name="node1",
+            consistency_level=None,
```

Comment:
Would it be possible to use server-mock tests instead? (check the mock_tests folder for examples)
These direct mock tests tend to break a lot. Happy to help, but we can also do it another time

Reply:
Let's chat about this next time you're available 🙂